lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -761,6 +761,26 @@ module Resources
           ]
         }
       },
+      {
+        'op': 'add',
+        'path': '/identifier/-',
+        'value': {
+          'type': {
+            'coding': [
+              {
+                'code': 'MR',
+                'system': 'http://hl7.org/fhir/v2/0203'
+              }
+            ]
+          },
+          'system': '2.16.840.1.113883.6.1000',
+          'value': 'THIS:IS:A:UNIQUE:IDENTIFIER',
+          'period': {
+            'start': '2018-01-02T00:00:00-05:00',
+            'end': '2022-01-02T00:00:00-05:00'
+          }
+        }
+      },
```

Comment:
Should we keep the example here and in the field specific examples the same?

Reply:
yea good idea changed [here](https://github.com/cerner/fhir.cerner.com/pull/555/commits/1530f2ab8911c92413e05c79ec4bf854aa3d1512#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL779) and [here](https://github.com/cerner/fhir.cerner.com/pull/555/commits/1530f2ab8911c92413e05c79ec4bf854aa3d1512#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dL87)